pypesto/sampling/sampler.py

```diff
@@ -126,9 +129,14 @@ def set_last_sample(self, sample: InternalSample):
             The sample that will replace the last sample in the chain.
         """
 
-    def make_internal(self):
+    def make_internal(self, temper_lpost: bool):
         """
         This function can be called by parallel tempering samplers during
         initialization to allow the inner samplers to adjust to them
         being used as inner samplers. Default: Do nothing.
+
+        Parameters
+        ----------
+        tamper_lpost:
```

Comment:
```suggestion
        temper_lpost:
```
correcting myself :see_no_evil: